binr/r2r/r2r.c

```diff
@@ -82,6 +84,10 @@ static bool r2r_chdir(const char *argv0) {
 #endif
 }
 
+static void r2r_test_run_unit() {
```

Comment:
`static void r2r_test_run_unit(void)`

Reply:
i fixed this in 10 other places inside run.c, thanks for pointing out